source/descriptions.cpp

```diff
@@ -103,6 +103,11 @@ string_view fountainOpen              = "King Zora starts as moved in both the c
                                         "adult eras. This also removes Ruto's Letter from\n"
                                         "the pool since it can't be used.";                //
 /*------------------------------                                                           //
+|         JABU-JABU            |                                                           //
+------------------------------*/                                                           //
+string_view jabuJabuCloseDesc         = "Jabu-Jabu's mouth stays closed until a fish is fed.";
```

Comment:
The `//` mark how long the text can be until it reaches the end of the screen, so at the moment the period will be on a new line. Can be fixed like this:
```suggestion
string_view jabuJabuCloseDesc         = "Jabu-Jabu's mouth stays closed until a fish is\n" //
                                        "fed.";                                            //
```